test/Datadog.Trace.Tests/HttpHeadersPropagatorTests.cs

```diff
@@ -28,12 +32,14 @@ public void WebRequest_InjectExtract_Identity()
         {
             const int traceId = 9;
             const int spanId = 7;
-            var headers = WebRequest.CreateHttp("http://localhost").Headers;
-            var context = new SpanContext(traceId, spanId);
+            const SamplingPriority samplingPriority = SamplingPriority.UserKeep;
 
-            headers.Inject(context);
+            WebHeaderCollection webHeaders = WebRequest.CreateHttp("http://localhost").Headers;
+            var context = new SpanContext(traceId, spanId, samplingPriority);
 
-            var resultContext = headers.Extract();
+            IHeadersCollection headersWrapper = HeadersFactory.Wrap(webHeaders);
+            headersWrapper.InjectSpanContext(context);
+            var resultContext = headersWrapper.ExtractSpanContext();
 
             Assert.Equal(context.SpanId, resultContext.SpanId);
             Assert.Equal(context.TraceId, resultContext.TraceId);
```

Comment:
Do we need to assert on sampling priority here?

Reply:
Good catch, thanks. Fixed in 4d8fc3a7d6c667da60c5ea64797b364302f2a935.